### PR TITLE
APP-298-6 - Redirection on an "in-progress" or "finalised" order

### PR DESCRIPTION
**Pull request by rcdtroly**
### Why:
* Current generated email from Club run `Update your order` will go to Cart page.
  Alt text
### What:
* Update redirection to Order page
### Refs:
* https://app.asana.com/0/939825968139797/1142183391173067

### CODE REVIEW

File: wp99234.php [hunk 1]
```diff
@@ -181,7 +181,7 @@ private function check_requirements() {
         private function define_constants() {
             $this->define( 'WP99234_HOST_IP', '103.18.108.127');
             $this->define( 'WP99234_PROTOCOL', 'https://');
-            $this->define( 'WP99234_DOMAIN', 'app.troly.io');
+            $this->define( 'WP99234_DOMAIN', 'stagingsubs.herokuapp.com');
```
Comment (JamesEmpireOne) [inline wp99234.php]: Should be `app.troly.io`!

File: includes/frontend/controllers/api/class-wp99234-api-orders.php [hunk 2]
```diff
@@ -144,40 +144,45 @@ function get( $route ){
...
             }
-            
+
+          }
+
+          // Redirect to order details page
+          if ($order_data = $order->get_data()) {
+            wp_redirect('/my-account/view-order/' . $order_data['id']);
+          } else {
+            wp_redirect(wc_get_cart_url());
```
Comment (JamesEmpireOne) [inline includes/frontend/controllers/api/class-wp99234-api-orders.php]: This line should redirect to the "my orders" area of WooCommerce

File: wp99234.php (hunk 1 above)
Comment (rcdtroly) [reply wp99234.php]: @JamesEmpireOne fixed

File: includes/frontend/controllers/api/class-wp99234-api-orders.php (hunk 2 above)
Comment (rcdtroly) [reply includes/frontend/controllers/api/class-wp99234-api-orders.php]: @JamesEmpireOne point to "my account" > "orders"

**Pull request merged**